### DISCUSSION TITLE
New release with updated Bouncy Castle?

### DISCUSSION THREAD

**Discussion by rdhzl** [pgpainless/pgpainless#447] [General]
Greetings:

 I noticed that the underlying bouncy castle version (1.77) has a few CVEs lodged against it that are resolved in 1.78. I also see that in your current, unpublished version you've got 1.78.1 as a dependency. I wanted to see if you were anticipating releasing the current iteration any time soon, or if it's worth considering subbing in 1.78.1 in the interim. Thank you!

**Comment by vanitasvitae** [comment]
Hey!
You are right, it might make sense to push another intermediate release with BC 1.78 until 1.79 lands. I'll give it a shot!

**Reply by rdhzl** [reply]
Thank you!

**Reply by vanitasvitae** [reply]
I just released PGPainless 1.7.0 which should be available on maven central soon :)

**Reply by vanitasvitae** [reply]
Whoopsie, Bumping a dependency caused a bug, so I quickly released PGPainless 1.7.1 too :)
